chain/sync.go

```diff
@@ -1054,14 +1054,15 @@ func (syncer *Syncer) checkBlockMessages(ctx context.Context, b *types.FullBlock
 		return xerrors.Errorf("failed to load base state tree: %w", err)
 	}
 
-	pl := vm.PricelistByEpoch(baseTs.Height())
+	nv := syncer.sm.GetNtwkVersion(ctx, b.Header.Height)
```

Comment:
I'm changing this from the base tipset's version to the block including the message's version which I think is the desired behavior.

Reply:
Good call out.  I checked this against mainnet and this code syncs past the calico upgrade (265060 to 265241 so far) so we should be all good.